photon-ml/src/main/scala/com/linkedin/photon/ml/data/LocalDataSet.scala

```diff
@@ -15,14 +15,14 @@
 package com.linkedin.photon.ml.data
 
 import java.util.Random
```

Comment:
@namitk - Namit, make sure you get the "Spark Style" setting from me or Alex to install in your IDEA. It has e.g. the formatting we want for the imports, and of course, a lot more. We try to standardize on that.